app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -548,6 +585,21 @@ class ChatActivity :
         this.lifecycle.addObserver(ChatViewModel.LifeCycleObserver)
     }
 
+    override fun onSaveInstanceState(outState: Bundle) {
+        if(currentlyPlayedVoiceMessage != null) {
+            // stores audio message ID and audio position
+            // so that can be restored in resumeAudioPlaybackIfNeeded method
```

Comment:
From my side, comments like this are not necessary.
The code itself has been implemented well so that no further explanation is necessary here.